source/network-security.txt

```diff
@@ -12,18 +12,137 @@ Network Security
    :depth: 1
    :class: onecol
 
-Intro statement
+By default, {+service+} secures your database deployments through the
+use of mandatory TLS/SSL connection encryption, and a VPC-first,
+minimum-access default policy. You can further configure these
```

Comment:
which "default policy" is meant? Did you mean access policy that allows minimum access based on some roles? Did you mean that on top of that access policy we recommend configuring vpc? 
I find this bit very densely packed, we could turn it into a simpler bullet list with links, I think. But, I am not sure this would not change the technical content. So please see what works -- these are just thoughts on how this reads, not a request for change. Hope you can find a middle ground.